import_email_events.py

```diff
@@ -0,0 +1,22 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL ENCODE lzo,
+    locale VARCHAR(40) ENCODE lzo,
+    domain VARCHAR(40) ENCODE lzo,
+    template VARCHAR(64) ENCODE lzo,
+    flow_id VARCHAR(64) DISTKEY ENCODE lzo
+"""
+
+COLUMNS = "type, locale, domain, template, flow_id"
+
+import_events.run(s3_prefix="fxa-email",
```

Comment:
The prefix needs to match as far as the hyphen immediately preceding the date in each file name, so this should be `fxa-email/events` in your case.

This fixes:

```
Traceback (most recent call last):
  File "./import_email_events.py", line 21, in <module>
    perm_columns=COLUMNS)
  File "/home/ec2-user/fxa-activity-metrics/import_events.py", line 225, in run
    unpopulated_days = get_unpopulated_days()
  File "/home/ec2-user/fxa-activity-metrics/import_events.py", line 161, in get_unpopulated_days
    if is_candidate_day(day) and not is_day_populated(day):
  File "/home/ec2-user/fxa-activity-metrics/import_events.py", line 148, in is_day_populated
    return bool(db.one(Q_CHECK_FOR_DAY.format(event_type=event_type, day=day)))
  File "/usr/lib/python2.7/site-packages/postgres/__init__.py", line 482, in one
    return cursor.one(sql, parameters, default)
  File "/usr/lib/python2.7/site-packages/postgres/cursors.py", line 105, in one
    out = self._some(sql, parameters, lo=0, hi=1)
  File "/usr/lib/python2.7/site-packages/postgres/cursors.py", line 127, in _some
    self.execute(sql, parameters)
  File "/usr/lib64/python2.7/site-packages/psycopg2/extras.py", line 288, in execute
    return super(NamedTupleCursor, self).execute(query, vars)
psycopg2.DataError: invalid input syntax for type date: "events-2017-03-20"
```